frontend/packages/kubevirt-plugin/integration-tests/tests/cdi-upload.scenario.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from 'path';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import {
+  createItemButton,
+  isLoaded,
+  saveChangesBtn,
+} from '@console/internal-integration-tests/views/crud.view';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/constants/common';
+import * as cdiUploadView from '../views/cdiUploadView';
+import { inputPVCName, inputPVCSize } from '@console/internal-integration-tests/views/storage.view';
+import {
+  selectItemFromDropdown,
+  storageclassDropdown,
+} from '@console/ceph-storage-plugin/integration-tests/views/pvc.view';
+
+describe('KubeVirt CDI Upload', () => {
+  const pvc = { name: 'upload-pvc', size: 1, unit: 'Gi', storageClassName: 'rook-ceph' };
+
+  it(
+    'ID() Upload data to CDI',
+    async () => {
+      const fileToUpload = '../views/cdiUploadView.ts';
+      const absolutePath = path.resolve(__dirname, fileToUpload);
+
+      await clickNavLink(['Storage', 'Persistent Volume Claims']);
+      await isLoaded();
+      await click(createItemButton);
+      await click(cdiUploadView.uploadCdiFormButton);
+
+      await browser.wait(
+        until.textToBePresentInElement(
+          $('.co-m-pane__heading'),
+          'Upload Data to Persistent Volume Claim',
+        ),
+      );
+
+      await inputPVCName.sendKeys(pvc.name);
+      await inputPVCSize.sendKeys(pvc.size);
+      await selectItemFromDropdown(pvc.storageClassName, storageclassDropdown);
+
+      // firefox needs the input to be shown
+      await browser.executeAsyncScript(function(callback) {
```

Comment:
do you have some more info on this? The input is visible by default, no?

Reply:
default is:
<img width="796" alt="Screen Shot 2020-08-19 at 12 38 48" src="https://user-images.githubusercontent.com/24938324/90618703-05590000-e219-11ea-96ff-ef6cb4ef270e.png">

firefox needs the input to be shown like this: (just for tests)
<img width="611" alt="Screen Shot 2020-08-19 at 12 38 59" src="https://user-images.githubusercontent.com/24938324/90618727-0ab64a80-e219-11ea-88c5-2209df9e5be3.png">

Reply:
Our tests is running against chrome only, it's can be removed if it's firefox specific.

Reply:
but developers can run this test locally with firefox?

Reply:
If it has no side affect  to run it with chrome, I'm okay with it.

Reply:
okay